src/bridge/cockpitpeer.c

```diff
@@ -394,9 +403,6 @@ cockpit_peer_dispose (GObject *object)
     on_other_closed (self->other, "terminated", self);
   g_assert (self->other == NULL);
 
-  self->inited = FALSE;
```

Comment:
Yes, it was causing trouble ... in general we don't want to reset the state completely on dispose. We still want to keep book-keeping data around, so we don't try to start up the peer yet again during shutdown.